### PR TITLE
[tensorflow, mxnet, pytorch ] | [build, test] | [sanity] ignore pillow issue for all existing py2 images

### PR DESCRIPTION
**Pull request by YYStreet**
*Issue #, if available:*
Ignore these known py2 issues.
```
---
-> pillow, installed 6.2.2, affected <7.0.0, id 38451
In libImaging/Jpeg2KDecode.c in Pillow before 7.0.0, there are multiple out-of-bounds reads via a crafted JP2 file. See: CVE-2020-10994.
--
-> pillow, installed 6.2.2, affected <6.2.3, id 38449
In libImaging/PcxDecode.c in Pillow before 6.2.3 and 7.x before 7.0.1, an out-of-bounds read can occur when reading PCX files where state->shuffle is instructed to read beyond state->buffer. See: CVE-2020-10378.
--
-> pillow, installed 6.2.2, affected <6.2.3, id 38450
In Pillow before 6.2.3 and 7.x before 7.0.1, there are two Buffer Overflows in libImaging/TiffDecode.c. See: CVE-2020-10379.
--
-> pillow, installed 6.2.2, affected <=7.0.0, id 38452
In libImaging/SgiRleDecode.c in Pillow through 7.0.0, a number of out-of-bounds reads exist in the parsing of SGI image files. See: CVE-2020-11538. Note that this is a different issue than CVE-2020-5311.
```
## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [tensorflow, mxnet, pytorch ] | [build, test] | [sanity] 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: mxnet/inference/docker/artifacts/config.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-vmargs=-XX:-UseContainerSupport
+vmargs=-XX:+UseContainerSupport -XX:InitialRAMPercentage=8.0 -XX:MaxRAMPercentage=10.0 -XX:-UseLargePages -XX:+UseG1GC -XX:+ExitOnOutOfMemoryError
```
Comment (TusharKanekiDey) [inline mxnet/inference/docker/artifacts/config.properties:1]: Is this change still needed ?

Comment (YYStreet) [reply mxnet/inference/docker/artifacts/config.properties]: Sorry. I was using this branch to test. Will remove it

File: test/dlc_tests/sanity/test_safety_check.py [hunk 2]
```diff
@@ -25,22 +25,30 @@
 IGNORE_SAFETY_IDS = {
     "tensorflow": {
         "training": {
-            "py2": ['37524'],
+            "py2": ['37524',
```
Comment (saimidu) [inline test/dlc_tests/sanity/test_safety_check.py:28]: Could you remove 37524 from all the lists?

Comment (YYStreet) [reply test/dlc_tests/sanity/test_safety_check.py]: Removed. Thanks

**Pull request merged**